smart-cards/src/main/java/gov/va/api/health/smartcards/WellKnownController.java

```diff
@@ -0,0 +1,32 @@
+package gov.va.api.health.smartcards;
+
+import gov.va.api.health.r4.api.information.WellKnown;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(
+    value = {"/.well-known/smart-configuration"},
```

Comment:
Should this be under `/r4/.well-known/...`?

Reply:
Nevermind. Keep this at top level, unless there's any differences between r4 and dstu2 versions (there shouldn't be). The unifier exposes the public endpoint at the correct endpoint.